pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```
@@ -127,6 +127,43 @@ spec:
             - name: socket-dir
               mountPath: /csi
         {{ end }}
+        {{ if .EnableCSIAddonsSideCar }}
+        - name: csi-addons
+          image: {{ .CSIAddonsImage }}
+          args :
+            - "--node-id=$(NODE_ID)"
+            - "--v=5"
+            - "--csi-addons-address=$(CSI_ADDONS_ENDPOINT)"
+            - "--controller-endpoint=$(POD_IP):4080"
```

```suggestion
            - "--controller-endpoint=$(POD_IP):4080"
```

Make the port configurable too.